packages/terra-i18n-plugin/tests/i18n-plugin-spec.test.js

```diff
@@ -69,7 +69,7 @@ describe('i18n-aggregator-plugin', () => {
         outputtedFileContent.push(content);
       });
       SpyOnConsoleWarn = spyOn(console, 'warn').and.callFake((message) => {
-        consoleMessage = message;
+        consoleMessages.push(message);
```

Comment:
In this before block, let's either reinitialize the consoleMessages to an empty array, or empty the consoleMessages array. We want to make sure all tests have a clean slate.

Reply:
Each describe block implements their own run of the plugin so the console logs are specific to each block.